web/client/epics/geoProcessing.js

```diff
@@ -589,12 +603,12 @@ export const runBufferProcessGPTEpic = (action$, store) => action$
                         }));
                     })
         );
-        const feature = sourceFeatureSelector(state);
+        const feature = transformLineToArc(sourceFeatureSelector(state));
```

Comment:
The fact that we have to transform the geometry in 8 different points of the code risk to make the feature not maintainable. Try to centralize this behavior in a single place (selector).
Try to do also the normalization once (labels, ids, managemant of measureId etc...). The geodesic tool should be as much transparent as possible to the special case of measure).

Reply:
sounds good, let me better see this